packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -6,18 +6,16 @@ import {
   BasicSpinner,
   useBufferState,
 } from '@openmsupply-client/common';
+import { ItemRowWithStatsFragment } from '@openmsupply-client/system';
```

Comment:
Request item edit modal main change is that the item fragment is coming from the item package, rather than the one generated by the request package